### PR TITLE
Update options-data.md

### PR DESCRIPTION
**Pull request by likev**
## Description of Problem
from Vue.js 2.x you can not just watch an expression 

## Additional Information
https://github.com/vuejs/vue/issues/844#issuecomment-271144849

### CODE REVIEW

File: src/api/options-data.md [hunk 1]
```diff
@@ -177,7 +177,7 @@
 
 - **Details:**
 
-  An object where keys are expressions to watch and values are the corresponding callbacks. The value can also be a string of a method name, or an Object that contains additional options. The component instance will call `$watch()` for each entry in the object at instantiation. See [$watch](instance-methods.html#watch) for more information about the `deep`, `immediate` and `flush` options.
+  An object where keys are data properties or computed properties to watch and values are the corresponding callbacks. The value can also be a string of a method name, or an Object that contains additional options. The component instance will call `$watch()` for each entry in the object at instantiation. See [$watch](instance-methods.html#watch) for more information about the `deep`, `immediate` and `flush` options.
```
Comment (bencodezen) [inline src/api/options-data.md:180]:
```suggestion
  An object where keys are reactive properties to watch and values are the corresponding callbacks. The value can also be a string of a method name, or an Object that contains additional options. The component instance will call `$watch()` for each entry in the object at instantiation. See [$watch](instance-methods.html#watch) for more information about the `deep`, `immediate` and `flush` options.
```

Comment (bencodezen) [reply src/api/options-data.md:180]: What do you think about this update @likev?

**Review by bencodezen — CHANGES_REQUESTED** [review]
What do you think about these updates?

File: src/api/options-data.md (hunk 1 above)
Comment (likev) [reply src/api/options-data.md:180]: good!

**Comment by likev** [comment]
> 
> 
> What do you think about these updates?

I think we can merge and close the pull

**Comment by skirtles-code** [comment]
I personally find this wording a little difficult to follow. The 'to watch' after the second dash throws me off. I tried reading it out loud and I don't seem to be able to put the emphasis in the right place.

File: src/api/options-data.md [hunk 2]
```diff
@@ -177,7 +177,7 @@
 
 - **Details:**
 
-  An object where keys are expressions to watch and values are the corresponding callbacks. The value can also be a string of a method name, or an Object that contains additional options. The component instance will call `$watch()` for each entry in the object at instantiation. See [$watch](instance-methods.html#watch) for more information about the `deep`, `immediate` and `flush` options.
+  An object where keys are reactive properties — examples include [data](/api/options-data.html#data-2) or [computed](/api/options-data.html#computed) properties — to watch and values are the corresponding callbacks. The value can also be a string of a method name, or an Object that contains additional options. The component instance will call `$watch()` for each entry in the object at instantiation. See [$watch](instance-methods.html#watch) for more information about the `deep`, `immediate` and `flush` options.
```
Comment (bencodezen) [inline src/api/options-data.md:180]:
```suggestion
  An object where keys are reactive properties to watch — examples include [data](/api/options-data.html#data-2) or [computed](/api/options-data.html#computed) properties — and values are the corresponding callbacks. The value can also be a string of a method name, or an Object that contains additional options. The component instance will call `$watch()` for each entry in the object at instantiation. See [$watch](instance-methods.html#watch) for more information about the `deep`, `immediate` and `flush` options.
```

**Comment by bencodezen** [comment]
@skirtles-code Good observation! What do you think of the movement of "to watch" earlier on to better separate out the concepts?

**Comment by skirtles-code** [comment]
I like it. Merging. Thank you both.

**Pull request merged**